src/client-python/reportclient/debuginfo.py

```diff
@@ -432,17 +432,18 @@ def download(self, files, download_exact_files=False):
 
         return RETURN_OK
 
-
```

Comment:
Something something two newlines after class definitions? https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Huh, didn't know about that one. Fixed'd.